### PR TITLE
remove `install-hook`

### PR DESCRIPTION
**Pull request by TApplencourt**
- Found a bug in `mpi` (didn't checked the correct ENV, fixed)

### CODE REVIEW

File: xprof/xprof.rb.in [hunk 1]
```diff
@@ -1,6 +1,5 @@
 #!/usr/bin/env ruby
 
-
```
Comment (TApplencourt) [inline xprof/xprof.rb.in:3]: >< thanks rubocopt

File: mpi/tracer_mpi_helpers.include.c [hunk 2]
```diff
@@ -7,7 +7,7 @@ static void _load_tracer(void) {
   void *handle = NULL;
   int verbose = 0;
 
-  s = getenv("LTTNG_UST_MPI_LIBMPI_LOADER");
+  s = getenv("LTTNG_UST_MPI_LIBMPI");
```
Comment (TApplencourt) [inline mpi/tracer_mpi_helpers.include.c:10]: It was a bug that we now catch

**Review by Kerilk — APPROVED** [review]
LGTM!

File: mpi/tracer_mpi_helpers.include.c (hunk 2 above)
Comment (Kerilk) [reply mpi/tracer_mpi_helpers.include.c:10]: Nice

**Pull request merged**